website/themes/uppy/layout/example.ejs

```diff
@@ -27,17 +27,18 @@
 
   <textarea id="console-log" class="Console"></textarea>
   <script>
-  console.log = (function (old_function, div_log) {
-    return function (text) {
-      old_function(text);
-
-      if (text !== (text + '')) {
-        text = JSON.stringify(text);
-      }
-
-      div_log.value = text + "\n" + div_log.value;
-    };
-  } (console.log.bind(console), document.getElementById("console-log")));
+    console.log = (function (log, container) {
+      return function (text) {
+        log(text);
+
+        if (text !== (text + '')) {
+          text = JSON.stringify(text);
+        }
+
+        container.value += text;
+        container.scrollTop = container.scrollHeight + '\n';
+      };
```

Comment:
isn’t `scrollTop` a number? how does `container.scrollHeight + '\n'` work?

Reply:
omg that `+ '\n'` was supposed to go on the line above :laughing:

Reply:
lol 😂 that thought crossed my mind later on briefly. I’ll update!